README.md

```diff
@@ -33,6 +33,85 @@ The plan is to use OCI projects and best of breed libraries for different aspect
 
 It is currently in active development in the Kubernetes community through the [design proposal](https://github.com/kubernetes/kubernetes/pull/26788).  Questions and issues should be raised in the Kubernetes [sig-node Slack channel](https://kubernetes.slack.com/archives/sig-node).
 
+## Getting started
+
+### Prerequisites
+`runc` is expected to be installed on the system. It is picked up as the default runtime by ocid.
```

Comment:
we could lookup in $PATH then - since there's already a --runtime flag where you can specify a path to runc